src/app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "player",
   "author": "sunbird",
-  "version": "5.0.0.4",
+  "version": "5.0.0",
```

Comment:
Version number is changes back to 5.0.0. It should be 5.0.0.5 right?

Reply:
semantic version will not take work in local so removed this
I can add the same if needed